content/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -12,110 +12,97 @@ description: >-
+Creating an account requires a username, an email address, and a password. To use Terraform Cloud, you must create an account through one of the following methods:
 
-If someone invited you to join an existing organization, the invitation email should include a sign-up link.
+- **Invitation Email:** When a user sends you an invitation to join an existing Terraform Cloud organization, the email includes a sign-up link. After you create an account, you will automatically join that organization and can begin using Terraform Cloud.
+- **Sign-Up Page:** For Terraform Cloud, go to [https://app.terraform.io/signup/account](https://app.terraform.io/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email.
 
-Alternately, you can create an account from Terraform Cloud's sign-up page. Navigate to the sign-up page via the link on the login page, or go directly to its URL:
+## Joining Organizations and Teams
 
-- For the SaaS version of Terraform Cloud, create a new account at <https://app.terraform.io/signup/account>.
-- For Terraform Enterprise, go to `https://<TFE HOSTNAME>/signup/account`.
-
-### After Creating an Account
-
-If you used an invitation link to create your account, you will automatically join that organization and can start using Terraform Cloud's features right away.
-
-If you created your account without an invitation, you will not initially belong to any organizations, and Terraform Cloud will display a page where you can create a new organization.
-
-![screenshot: the new organization page](/img/docs/org-new.png)
-
-- If you are the first Terraform Cloud user in your organization, use this page to create your Terraform Cloud organization. See [Organizations][] for more information.
-- If you intend to join an existing organization, do not create a new one; instead, send your email address to one of your organization's owners and ask them to invite you. Once you receive and accept the emailed invitation, you can reload the page to begin using Terraform Cloud.
-
-## Team and Organization Membership
-
-To join an organization, you must be invited by an owner of that organization. In order to have any permissions with that organization, an owner will need to add you to one or more teams. For more information, see [Organization Settings: Users][invite] and [Managing Team Membership](/cloud-docs/users-teams-organizations/teams#managing-team-membership).
+An organization owner must [invite you to join an organization](/cloud-docs/users-teams-organizations/organizations#users). After you join, the organization owner must also [add you to one or more teams](/cloud-docs/users-teams-organizations/teams#managing-team-membership).
```

Comment:
```suggestion
An organization owner must [invite you to join their organization](/cloud-docs/users-teams-organizations/organizations#users). After you join, the organization owner must also [add you to one or more teams](/cloud-docs/users-teams-organizations/teams#managing-team-membership).
```

I know this is obvious, but I feel that this change gets rid of a small understanding loophole where users might believe that an invite from any org owner could get them access to other orgs which aren't associated with the owner who invited them.

Reply:
yes! thank you!